client/my-sites/site-settings/taxonomies/taxonomy-card.jsx

```diff
@@ -50,7 +50,9 @@ const TaxonomyCard = ( {
 			<h2 className={ classes }>{ labels.name }</h2>
 			{ ! isLoading && (
 				<div className="taxonomies__card-content">
-					<Gridicon icon="tag" size={ 18 } /> { count } { labels.name }
+					<Gridicon icon="tag" size={ 18 } />
+					&nbsp;
+					{ count } { count === 1 ? labels.singular_name : labels.name }
```

Comment:
Nitpick: Maybe the label ternary expression can be extracted into a variable for better auto formatting and to avoid the need of `&nbsp;`, i.e.
```
const label = count === 1 ? labels.singular_name : labels.name;
...
<Gridicon icon="tag" size={ 18 } /> { count } { label }
```

Reply:
Thanks, Yuliyan! 😀  That is a good point, I was already doubting that a little bit actually 🤔  I actually considered what you say, for example putting this above the return statement:

```js
const labelAfterCount = count === 1 ? labels.singular_name : labels.name;
```

But then I felt it would be more correct to also check for `isLoading` there (because both `count` and `labels` would be empty if true). That would make it more verbose, and because of that, I thought it would be a bit cleaner to put it in the return statement, because it would already be behind this `! isLoading` check.

I do realize that `labelAfterCount` would just be `undefined` and unused if `isLoading` is true (because `labels.name` would be `undefined`) 🙂 

Knowing this, would you extract the variable in this case (with or without the `isLoading` check)? I know these are very small things, but I'd love to learn how you would do this! 😀 Thanks!

Reply:
I hadn't noticed the `isLoading` condition used there, but in this case what I suggested above might make the code a bit more confusing to read.

I'd personally still try to avoid the use of `&nbsp;`. So adding the `isLoading` flag seems like a good solution, e.g.:

```js
let label;

if ( ! isLoading ) {
	label = count === 1 ? labels.singular_name : labels.name;
}
```

Reply:
Thanks for the suggestion, done!